packages/react-components/src/components/kpi/types.ts

```diff
@@ -1,7 +1,7 @@
 import type { WidgetSettings } from '../../common/dataTypes';
 
 export type KPIProperties = WidgetSettings & {
-  settings: KPISettings;
+  settings?: Partial<KPISettings>;
 };
```

Comment:
Non-blocking question: Why do KPI settings need to be made optional? Why not set them as optional from the start?

Reply:
good catch, i was not quite doing what I intended on doing - slightly revised but kept the partial

Reply:
Nice! Thank you!